modules/restoring-the-3scale-operator-secrets-and-apimanager.adoc

```diff
@@ -3,18 +3,18 @@
 //included in backing-up-and-restoring-3scale-by-using-oadp.adoc assembly
 
 
-[id="restoring-the-secrets-and-apimanager_{context}"]
-= Restoring the secrets and APIManager
+[id="restoring-the-3scale-operator-secrets-and-apimanager_{context}"]
+= Restoring the 3scale Operator, secrets, and APIManager
 
-You can restore the Secrets and APIManager by using the following procedure.
+You can restore the 3scale Operator resources, and `Secret` and APIManager custom resources (CR) by using the following procedure.
 
 .Prerequisites
 
 * You backed up the 3scale Operator.
-* You backed up `mysql` and Redis databases.
+* You backed up the MySQL and Redis databases.
```

Comment:
```suggestion
* You backed up the MySQL and Redis databases.
```
Again, are these required databases, supported databases, just examples for any customer databases?